data/saas/config/segment_config.yml

```diff
@@ -166,7 +166,10 @@ saas_config:
       - name: workspace_name
         connector_param: workspace
       - name: user_id
-        identity: email
+        references:
+          - dataset: <instance_fides_key>
+            field: traits.user_id
```

Comment:
I chose to use the traits, but it could also be that we use `external_ids` however it will end up dropping extra requests so decided against it for the time being